apps/wing/src/wingc.ts

```diff
@@ -25,12 +25,14 @@ export interface WingCompilerLoadOptions {
 
 export async function load(options: WingCompilerLoadOptions) {
   const preopens = {
+    "/": "/",
     // .jsii access
     [WINGSDK_MANIFEST_ROOT]: WINGSDK_MANIFEST_ROOT,
```

Comment:
Even though we add "/", adding other explicit preopens is useful in the case of Windows and if any files we need are in drives other than C.